neonvm/runner/main.go

```diff
@@ -277,11 +281,19 @@ func createISO9660runtime(
 				mounts = append(mounts, fmt.Sprintf(`/neonvm/bin/mount %s $(/neonvm/bin/blkid -L %s) %s`, opts, disk.Name, disk.MountPath))
 				// Note: chmod must be after mount, otherwise it gets overwritten by mount.
 				mounts = append(mounts, fmt.Sprintf(`/neonvm/bin/chmod 0777 %s`, disk.MountPath))
+				diskCnt += 1
+			case disk.RawDisk != nil:
+				device := fmt.Sprintf("/dev/vd%s", string(rune(int('a')+diskCnt)))
```

Comment:
This seems fragile (smells like multiple sources of truth). Any ideas for ways around it?

Reply:
Yep it's true, but I could not find a good way of doing an 1v1 mapping between serial number and `vd*`. The way it works is that we first guess where the raw block device would be, i.e., vde, and then use the utility tools to verify if it matches the serial number.

Reply:
How about with smth like `lsblk -o "serial,name" -p`? Or perhaps look it up how `lsblk` does it and reimplement the same way.